shared/listparser.go

```diff
@@ -51,6 +51,7 @@ type ListMetaData struct {
 	ListType       string // https://developer.mozilla.org/en-US/docs/Web/CSS/list-style-type
 	DigestInterval string
 	Email          string
+	InlineContent  string // allows content inlining to be disabled in feeds.sh emails
```

Comment:
This has to be hooked up to [TokenToMetaField](https://github.com/picosh/pico/blob/0d7a17435d3cf31e28ee9b1bb796a581d0582ae9/shared/listparser.go#L100) and you can make it a `bool`, set the default to true, and perform proper type checks on it.